### PR TITLE
DBAAS-2875: Update public API with the missing config options

### PR DESCRIPTION
**Pull request by shabtaisharon**

### CODE REVIEW

**Comment by ChiefMateStarbuck** [comment]
!deploy

**Comment by shabtaisharon** [comment]
Hi @ChiefMateStarbuck,
Can you please help me understand why the PR validation failed?

File: specification/resources/databases/models/mysql.yml [hunk 1]
```diff
@@ -105,3 +162,25 @@ properties:
...
+      Limits the size of internal in-memory tables. Also set
+      max_heap_table_size. Default is 16777216 (16M)
+    type: integer
+    minimum: 1048576
+    maximum: 1073741824
+    example: 16777216
+  slow_query_log:
+    description: >-
+      Slow query log enables capturing of slow queries. Setting slow_query_log
+      to false also truncates the mysql.slow_log table. Default is off
```
Comment (yGabriel) [inline specification/resources/databases/models/mysql.yml:176]:
```suggestion
      When enabled, captures slow queries. When disabled, also
      truncates the mysql.slow_log table. Default is false.
```

File: specification/resources/databases/models/mysql.yml [hunk 2]
```diff
@@ -105,3 +162,25 @@ properties:
...
+    type: boolean
+    example: true
+  long_query_time:
+    description: >-
+      The slow_query_logs work as SQL statements that take more than
+      long_query_time seconds to execute. Default is 10s
```
Comment (yGabriel) [inline specification/resources/databases/models/mysql.yml:182]:
```suggestion
      The time, in seconds, for a query to take to execute before 
      being captured by slow_query_logs. Default is 10 seconds.
```

**Review by yGabriel — COMMENTED** [review]
Left some in-line edits!

File: specification/resources/databases/models/mysql.yml [hunk 3]
```diff
@@ -25,9 +25,39 @@ properties:
...
+    type: integer
+    minimum: 0
+    maximum: 16
+    example: 3
+  innodb_ft_server_stopword_table:
+    description: >-
+      The InnoDB FULLTEXT index stopword
+      list for all InnoDB tables.
+    type:
+      - 'null'
```
Comment (ChiefMateStarbuck) [inline specification/resources/databases/models/mysql.yml:53]:
```suggestion
```
`type` should only be `- string`

**Review by ChiefMateStarbuck — CHANGES_REQUESTED** [review]

File: specification/resources/databases/models/mysql.yml (hunk 3 above)
Comment (shabtaisharon) [reply specification/resources/databases/models/mysql.yml]: Thank you!

**Review by ChiefMateStarbuck — APPROVED** [review]

**Pull request merged**